File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/details/add/ICreateConstellationView.java
package de.bitb.astroskop.ui.main.profiles.details.add;

import de.bitb.astroskop.model.Constellation;
import de.bitb.astroskop.ui.base.IBaseView;

interface ICreateConstellationView extends IBaseView{
    void finishConstellation(Constellation constellation);
}

File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/details/add/ICreateConstellationView.kt
package de.bitb.astroskop.ui.main.profiles.details.add

import de.bitb.astroskop.model.Constellation
import de.bitb.astroskop.ui.base.IBaseView

internal interface ICreateConstellationView : IBaseView {
    fun finishConstellation(constellation: Constellation)
}
